### ISSUE TITLE
Block list editor updates not deployed correctly

### ISSUE THREAD

**Issue by AllanKirk** [umbraco/Umbraco.Cloud.Issues#301]
There is a number of problems when I update a block list editor and try to deploy the changes to the next server in the chain.

1. I added some thumbnails to the blocks, but they were not deployed. The images themselves were deployed just fine, just not the settings on the block list.
2. When I reorder blocks in the block list, it is not deployed.
3. If I remove a block from a block list, it is not deployed.

Basically any time I make a change to the Block List, I have to go to each server in the chain and make the same changes manually.

**Comment by MMasey** [comment]
Just chiming in to say that I'm also experiencing this issue.

Thanks
